book/src/kernel/linux-compatibility/syscall-flag-coverage/process-and-thread-management/README.md

```diff
@@ -3,7 +3,7 @@
 <!--
 Put system calls such as
 fork, vfork, clone, execve, exit, exit_group, wait4, waitid,
-getpid, getppid, gettid, setuid, setgid, getuid, getgid, and prctl
+getpid, getppid, gettid, setuid, setgid, getuid, getgid, prctl, ioprio_get, and ioprio_set
```

Comment:
> Put system calls such as

Note the "such as" in the sentence. The list is inexhaustive. So you don't have to keep updating the list.